store/index.js

```diff
@@ -35,17 +35,20 @@ export const state = () => ({
         "Age": {
 
             componentName: "annot-continuous-values",
-            explanation: "This is an explanation for how to annotate age."
+            explanation: "This is an explanation for how to annotate age.",
+            identifier: "nb:hasAge"
```

Comment:
The term to use here would be for the class "Age", not the property "hasAge", so `nb:Age` instead of `nb:hasAge`. Just pinging @alyssadai here quickly to see if there is a nice place where this is listed - and if there isn't, we might want to consider one?

Reply:
Yep - I think this page https://www.neurobagel.org/documentation/dictionaries/#age has the up-to-date info for this variable 🙂